### PR TITLE
Update translation_ko.ts

### PR DESCRIPTION
**Pull request by sople1**
Hi, I want to request for update Korean Translation file.
Have a nice day! :3

### CODE REVIEW

**Comment by bovirus** [comment]
@procount 

Please merge the translation and I will check the end punctation with .ts/po tools.

File: recovery/translation_ko.ts [hunk 1]
```diff
@@ -872,7 +873,7 @@ Please provide it locally or connect to the internet.</source>
 See debug log for details.</source>
         <oldsource>OS(es) Downloaded with errors.
 See debug log for details</oldsource>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
Comment (lurch) [inline recovery/translation_ko.ts:876]: Empty translation here - was that deliberate?

**Comment by bovirus** [comment]
@lurch 

If you merge this PR I can make a statistics about all issues pending for this language.

**Comment by lurch** [comment]
> I will check the end punctation

Is that a Western-language assumption? Are there some languages which don't use a full-stop at the end of sentences? Or is ".ts/po tools" clever enough to account for those differences?
(I did a quick search and found e.g. https://en.wikipedia.org/wiki/Full_stop#Full_stops_in_other_scripts )

**Comment by lurch** [comment]
> @lurch If you merge this PR...

Only @procount is able to do that :slightly_smiling_face: 

But if you want to analyse this PR yourself pre-merge, you can simply clone https://github.com/sople1/pinn/tree/trans-ko-190711 and run your tools on that.

**Comment by bovirus** [comment]
@lurch 

The translation is now fully translated (475/475  strings). 
Ant that it's more important,
In this translation there are many end punctation mismatch.
I will fix when the translation will merge.
In the translation in the same similar context sometimes there is a "." at the end and sometimes not.
And it didn't match the sources.
Then I beleive that is due a  standard.

I suggest to all translator to use QT Linguist 4.8 and enable all check (end punctation/accelerator/etc).

I can provide the QTLinguist 4.8 standalone without installer.

File: recovery/translation_ko.ts (hunk 1 above)
Comment (sople1) [reply recovery/translation_ko.ts:876]: Oh, what the... thanks for your check. I will change it now

**Comment by sople1** [comment]
Oh, I found errors from my translated file so I will re-request after next commit.
Sorry for mistake. and thanks for your reviews :)

**Comment by sople1** [comment]
@bovirus 

about punctuation
You say that "same similar context sometimes there is a "." at the end and sometimes not".
It is difficult things sometime.
In Korean, 
sentence using punctuation means talk, say.
sentence not using punctuation means order, warning.
so if all sentences of translated have same punctuation with original translated, (although the sentence communicate to Korean users,)  it will look harsh to the users.
And I now using QT Linguist 5.12.4. I know that some punctuations not matched with originals.

Well, I don't know what is good for this project, but I think that better translation is user friendly thing.

**Comment by procount** [comment]
@sople1 
Unfortunately, I caused a lot of translation problems by cutting and pasting some documentation from the website documentation into the tooltips without taking enough care. My apologies.
The problems were: 
- Adding spaces to the beginning of sentences
- Missing a full-stop at the end of a sentence
- Adding spurious HTML code
- Splitting sentences into specific parts due to arguments, which may not make sense in other languages.

I think all but the last are now fixed in the original source, but they need to be fixed in the translations, which is what @bovirus has tried to sort out for me.

Of course, adding a full-stop at the end of every sentence is good in a latin language for consistency, but if it does not make sense in Korean, then please adjust as necessary. IMHO, friendly is better.
Thanks.

**Comment by sople1** [comment]
@procount 

I think sorry for no contributing for translating format strings on source code... TT
(Cause by my main language is PHP, Python... (web based) but not C++, Qt...(app based))
If can, I will try to request that format strings often....

Thnaks for your message and work. Have a nice day! :D

**Comment by procount** [comment]
It's not a problem. I can do the source file mods (with the help of Lurch).
Thank you for your contribution.

**Comment by bovirus** [comment]
@sople1 

I made all checks and fixes about start/end pucation about your KO langauge file.
Please check here if it's OK.

https://github.com/procount/pinn/pull/295

To translate .ts file could be better to use QTLinguist.

**Pull request closed without merging**